packaging/ansible-runner-service-project/project/roles/ovirt-host-enroll-certificate/tasks/main.yml

```diff
@@ -5,3 +5,10 @@
     - vmconsole
     - ovn
     - vnc
+
+- name: Update certs permissions
+  file:
+    path: '{{ ovirt_pki_dir }}'
+    mode: u=rwx,g=r,o=r
+    recurse: yes
```

Comment:
Ansible documentation says about `recurse`: This applies only when `state` is set to `directory`.  Does this actually apply recursively then?